ci/steps/publish-npm.sh

```diff
@@ -59,11 +59,11 @@ main() {
   # This string is used to determine how we should tag the npm release.
   # Environment can be one of three choices:
   # "development" - this means we tag with the PR number, allowing
-  # a developer to install this version with `yarn add code-server@<pr-number>`
+  # a developer to install this version with `npm install -g code-server@<pr-number>`
```

Comment:
I think in this file, we don't need `-g` since before we were telling folks to install as a dependency (notice we don't use `yarn global add`). Guessing this was a find and replace though

Reply:
Ah damn yes, my bad! Great catch!